src/DI/ContainerBuilder.php

```diff
@@ -411,6 +417,17 @@ private function resolveImplement(ServiceDefinition $def, $name)
 	}
 
 
+	private function getTypeHintForParameter(\ReflectionParameter $param)
+	{
+		if (PHP_VERSION_ID >= 70000 && $param->hasType()) {
+			$type = $param->getType();
+			return $type->isBuiltin() ? (string) $type : '\\' . $type;
+		} else {
+			return $param->isArray() ? 'array' : PhpReflection::getPropertyType($param);
```

Comment:
What about it? Callable is considered a built-in type by `ReflectionType::isBuiltin`.